tests/ip/test_mgmt_ipv6_only.py

```diff
@@ -64,12 +64,23 @@ def log_eth0_interface_info(duthosts):
 
 def log_tacacs(duthosts, ptfhost):
     for duthost in duthosts:
+        # Print debug info for ipv6 pingability
         ptfhost_vars = ptfhost.host.options['inventory_manager'].get_host(ptfhost.hostname).vars
         if 'ansible_hostv6' in ptfhost_vars:
             tacacs_server_ip = ptfhost_vars['ansible_hostv6']
             ping_result = duthost.shell(f"ping {tacacs_server_ip} -c 1 -W 3", module_ignore_errors=True)["stdout"]
             logging.debug(f"Checking ping_result [{ping_result}]")
 
+        # Print debug info for mgmt interfaces and forced mgmt routes
+        mgmt_interface_keys = duthost.command("sonic-db-cli  CONFIG_DB keys 'MGMT_INTERFACE|eth0|*'")['stdout']
```

Comment:
Will you print other MGMT_INTERFACE entries? Can we not assume the port name is always `eth0`?

Reply:
Updated